components/06-examples/checken.config.yml

```diff
@@ -1,18 +1,60 @@
+notes: |
+  De aside met de checklist schuift mee met het formulier, tot hij een punt bereikt waar hij niet verder MAG scrollen.
```

Comment:
Deze _notes_ gaan over het [progress block component](https://dso-toolkit.nl/7.3.0/components/detail/progress-block.html). Maar in dit PR haal je het progress block component juist weg - deze notes zijn dus niet nodig.

Daarbij: Wordt het progress block component nog wel gebruikt?